ui/page.go

```diff
@@ -669,6 +678,72 @@ func (page pageCommon) SelectedAccountLayout(gtx layout.Context) layout.Dimensio
 	return card.Layout(gtx, selectedDetails)
 }
 
+type SubPage struct {
+	title      string
+	walletName string
+	back       func()
+	body       layout.Widget
+}
+
+func (page pageCommon) SubPageLayout(gtx layout.Context, sp SubPage) layout.Dimensions {
+	if page.subPageInfoButton.Button.Clicked() {
+		go func() {
+			page.modalReceiver <- &modalLoad{
+				template:   InfoTemplate,
+				title:      "Verify message",
+				cancel:     page.closeModal,
+				cancelText: "Got it",
+			}
+		}()
+	}
```

Comment:
From what i understand about this function, its supposed to used by all the subpages. However whenever the `subPageInfoButton` info button is clicked, it will only render the verify message modal template. I believe this should be made flexible.